### ISSUE TITLE
How to install the new scraper

### ISSUE THREAD

**Issue by MaH-s9UFaXPEKdLafMHkSGgWUAupTA1xj78S9C7** [DIGITALCRIMINALS/OnlyFans#876]
Anyone who wouldn't mind teaching me how to install Poetry step-by-step and then this scraper following it?
I've tried looking for tutorials online, but I can't find any with the newest Poetry, so sadly I have no idea how to do it.
I hope one of you can help me out. Sorry for being so bad at it.

**Comment by CannotTouch** [comment]
He have insert the step-by-step link in the istruction in the first page:
https://github.com/DIGITALCRIMINALS/OnlyFans#installation-commands

**Comment by MaH-s9UFaXPEKdLafMHkSGgWUAupTA1xj78S9C7** [comment]
Thanks.
I did follow that and I seemed to get Poetry installed, but I cannot add it to PATH. I also tried using the full path (copy-paste), but it still doesn't work.

**Comment by neil-ui** [comment]
I found updating the path according to these instructions
https://www.geeksforgeeks.org/pythonpath-environment-variable-in-python/
Worked for me, I added [stuff]\Roaming\Python\Scripts and it started working

**Comment by MaH-s9UFaXPEKdLafMHkSGgWUAupTA1xj78S9C7** [comment]
Thank you very much.
It still doesn't work sadly, but that was at least the kind of step-by-step guide I hoped for, which I couldn't find myself.

This is the messages I get, along with the error message in the end:
Poetry (1.4.0) is installed now. Great!

To get started you need Poetry's bin directory (C:\Users\User\AppData\Roaming\Python\Scripts) in your `PATH`
environment variable.

Alternatively, you can call Poetry explicitly with `C:\Users\User\AppData\Roaming\Python\Scripts\poetry`.

You can test that everything is set up by executing:

`poetry --version`

PS C:\Users\User> poetry --version
poetry : The term 'poetry' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
 the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ poetry --version
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (poetry:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException


Are you able to see what I do wrong and what can I do to fix it?

**Comment by MaH-s9UFaXPEKdLafMHkSGgWUAupTA1xj78S9C7** [comment]
Ok, so I seemed to have gotten a bit further. Seems my computer installed Python in another directory than the one the step-by-step guide listed. I cut and pasted the poetry script from its' original location to the other Python location and now I at least got the version name to pop up. However, now I get these error messages:
1. D:\OnlyFans>python updater.py
Traceback (most recent call last):
  File "D:\OnlyFans\updater.py", line 8, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
2. D:\OnlyFans>poetry run python start_us.py
Creating virtualenv ultima-scraper-CglJ4UwT-py3.11 in C:\Users\User\AppData\Local\pypoetry\Cache\virtualenvs
Traceback (most recent call last):
  File "D:\OnlyFans\start_us.py", line 8, in <module>
    from ultima_scraper_api.helpers.main_helper import OptionsFormat
ModuleNotFoundError: No module named 'ultima_scraper_api'

I made a clean install (git clone) using this: https://github.com/DIGITALCRIMINALS/OnlyFans.git

Is anyone able to see, what I am doing wrong? Also is will it cause an issue that I cut and pasted the poetry script to another folder to make it work?

**Comment by MaH-s9UFaXPEKdLafMHkSGgWUAupTA1xj78S9C7** [comment]
Cool. Thank you so much both of you. That seems to have fixed it.
Anything I should be aware of that is new and special? Just so I don't inadvertently delete or change something in my old files and lose any content previously downloaded? Also, do you recommend using Powershell or CommandPrompt with this?

**Comment by DIGITALCRIMINALS** [comment]
> Cool. Thank you so much both of you. That seems to have fixed it. Anything I should be aware of that is new and special? Just so I don't inadvertently delete or change something in my old files and lose any content previously downloaded? Also, do you recommend using Powershell or CommandPrompt with this?

Powershell

**Comment by MaH-s9UFaXPEKdLafMHkSGgWUAupTA1xj78S9C7** [comment]
Ok, thanks.
Why is Powershell better than CommandPrompt though?
Is there a way to make it work, just by clicking the thumbnails as it did with the old one or is it necessary to go through commandprompt/powershell each time now to run the program?

I just tried updating it, but it gave this error:
Traceback (most recent call last):
  File "D:\OnlyFans\updater.py", line 8, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

Also is the blacklist function broken, because I tried to write it in the place for it, same as I did before. However now, it doesn't work at all. Both with multiple blacklists as well as a single one.
